pkg/skuba/actions/addon/upgrade/apply.go

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package addons
+
+import (
+	"fmt"
+
+	"github.com/SUSE/skuba/internal/pkg/skuba/addons"
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubeadm"
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubernetes"
+	"github.com/SUSE/skuba/internal/pkg/skuba/upgrade/addon"
+	"github.com/SUSE/skuba/pkg/skuba"
+	"github.com/pkg/errors"
+)
+
+// Apply implements the `skuba addon upgrade apply` command.
+func Apply() error {
+	fmt.Printf("%s\n", skuba.CurrentVersion().String())
```

Comment:
Is this intended or a leftover? :)

Reply:
Intended: I've seen it on `addon upgrade plan`, in `node upgrade apply` and so on, so I just assumed that this was a common thing to do. If this is not the case I can remove this line :+1:

Reply:
oh i think it's a leftover and we forgot to remove it, i guess it needs to be dropped also on the other functions

Reply:
I've added a commit to drop all of them in https://github.com/SUSE/skuba/pull/733 - so I guess you can remove this one here too

Reply:
Done